src/file.js

```diff
@@ -16,6 +17,17 @@ export function listLocalFiles (root) {
     nodir: true,
     ignore: getIgnoredPaths(root),
   })
+  .then(files =>
+    Promise.all(
+      files.map(file => bbStat(path.join(root, file))
+      .then(stats => ({file, stats})))
+    )
+  )
+  .then(filesStats =>
+    filesStats.reduce((acc, {file, stats}) => {
+      return stats.size > 0 ? [...acc, file] : acc
```

Comment:
Depending on how this is compiled by babel, it might get very inefficient. Might be better to `push` and `return` to avoid creating arrays. But *maybe* it does that already. :shrug: